dnsimple/dnsimple-api-examples
# New use case sample app to issue certificates

The sample application handles the issuance of certificates for a domain, with the addition of being able to handle webhooks.


## QA

1. Follow the readme to start up the example application.
2. Send a request to request a certificate for one of your domains (you can get the ngrok url from http://localhost:4040/status):
```bash
export NGROK_URL=''
curl --location --request POST "${NGROK_URL}/dnsimple/issue_certificate" \
--header 'Content-Type: application/json' \
--data-raw '{
    "domain": "<domain_name>",
    "name": "w5",
    "auto_renew": false
}'
```
3. Expect to issue a LetsEncrypt certificate for the requested domain.
4. Expect to see events being received at http://localhost:4040/inspect/http
